client/my-sites/plans/controller.jsx

```diff
@@ -74,18 +75,17 @@ module.exports = {
 		analytics.tracks.recordEvent( 'calypso_plans_view' );
 		analytics.pageView.record( analyticsBasePath, analyticsPageTitle );
 
-		ReactDom.render(
-			<ReduxProvider store={ context.store }>
-				<CartData>
-					<Plans
-						sites={ sites }
-						onSelectPlan={ handlePlanSelect }
-						plans={ plans }
-						context={ context }
-						destinationType={ context.params.destinationType }/>
-				</CartData>
-			</ReduxProvider>,
-			document.getElementById( 'primary' )
+		renderWithReduxStore(
+			<CartData>
+				<Plans
+					site={ site }
```

Comment:
Again you're right! Now I remember why we had this.
Ok I have no choice but to revert most of the changes and find another way :(